Jenkinsfile

```diff
@@ -33,9 +33,29 @@ node {
 
     if(env.BRANCH_NAME == "master") {
       stage('Publish Gem') {
-        sshagent(['govuk-ci-ssh-key']) {
-          govuk.runRakeTask("publish_gem --trace")
+        def version = sh(
+          script: /ruby -e "puts eval(File.read('govuk_navigation_helpers.gemspec'), TOPLEVEL_BINDING).version.to_s"/,
```

Comment:
Does anyone have any bright ideas on this? I think it's a bit weird to be running Ruby inside the shell inside Groovy (there's just so many layers it's kind of painful), but I can't think of a better way of getting at the version number.